ksql-functional-tests/src/test/resources/query-validation-tests/simple_agg.json

```diff
@@ -0,0 +1,19 @@
+{
+  "tests": [
+    {
+      "name": "simple_agg",
```

Comment:
Did you mean to check this in?  `simple_agg` : there's no aggregation here...

Reply:
deleted